### PR TITLE
Update NOTICE

### LINKED ISSUE #1629: chore: Update NOTICE year to 2023
This line should be updated

https://github.com/apache/apisix-ingress-controller/blob/4c0535ba929c64ed9f9ac603300156879fe79e26/NOTICE#L2

### PR DESCRIPTION
**Pull request by Rishav1707**
### Type of change:

- [x] New feature provided

### What this PR does / why we need it:

closes #1629

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Apache APISIX
-Copyright 2019-2022 The Apache Software Foundation
+Copyright 2023 The Apache Software Foundation
```
Comment (tao12345666333) [inline NOTICE:2]: please change this to  `2019-2023`

Thanks!

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

File: NOTICE (hunk 1 above)
Comment (Rishav1707) [reply NOTICE]: Done👍🏻

**Review by tao12345666333 — APPROVED** [review]

**Review by lingsamuel — APPROVED** [review]

File: NOTICE (hunk 1 above)
Comment (Rishav1707) [reply NOTICE]: rishavraj1707music@gmail.com

Comment (KID-G) [reply NOTICE]: > [rishavraj1707music@gmail.com](mailto:rishavraj1707music@gmail.com)

Ok, please check your email.

Comment (Rishav1707) [reply NOTICE]: Got it🙌🏻

**Pull request merged**